### ISSUE TITLE
bug: do not fail on empty session data string (sessoindata=="")

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2367]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.27.8](https://github.com/open-wa/wa-automate-nodejs/compare/4.27.7...4.27.8) (2021-12-22)

- Release 4.27.8 [`39b1f0e`](https://github.com/open-wa/wa-automate-nodejs/commit/39b1f0e374a961f5b5e1ce8f0b62827cbaa851c2)
- 🐛 fix: #2367 [`92cf311`](https://github.com/open-wa/wa-automate-nodejs/commit/92cf311ff70f5a5bafe1f78f417d0620e6d9e390)
- updated types-only package [`8cce0de`](https://github.com/open-wa/wa-automate-nodejs/commit/8cce0de404192ac0323c569dad5c9f5826863f1e)
